source/concepts/applications.rst

```diff
@@ -208,18 +152,19 @@ If they are available, a client can choose to represent them as they choose.
 
 .. note::
 
-   We assume that the points defining the play area are on the floor (Y=0).
-   However, nothing forces a client to send them a such.
+   Typically we assume that the points defining the play area are on the floor
+   (Y=0), but this is not required.
 
 .. _radial-orient:
 
 Radial orient
 ~~~~~~~~~~~~~
 
-A server can, optionally, implement the radial orient feature as a command on
-the :ref:`command service <command-service>`. The radial orient command places
-all the avatars on a circle around the origin of the server space by
-setting a :ref:`user origin <user-origin-description>` for each avatar.
+The radial orient feature is an command optionally implemented on the
+:ref:`command service <command-service>`. This command suggests how clients
+should position their avatars relative to server space such that all clients
```

Comment:
Is it only avatars or also the user's play space?

Reply:
It's now "This command suggests how clients should position their client space (and hence avatars)"